### PR TITLE
chore: let Dependabot open up to 10 PRs

### PR DESCRIPTION
**Pull request by bklebe**
I find it mildly annoying to have to manually pull in more updates every 5 PRs.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,3 +4,4 @@ updates:
   directory: "/"
   schedule:
     interval: daily
+  open-pull-requests-limit: 10
```
Comment (digitalcora) [inline .github/dependabot.yml:7]: ⛔❓

**Review by digitalcora — APPROVED** [review]
I'm fine with this, but if we were doing a "mass update" (i.e. there were ≥ `limit` PRs open), wouldn't we be doing the package updates locally anyway and thus know all the ones that were outdated?

File: .github/dependabot.yml (hunk 1 above)
Comment (bklebe) [reply .github/dependabot.yml:7]: Sorry, I don't follow?

Comment (digitalcora) [reply .github/dependabot.yml:7]: Oh, it only displays it in the main "Files changed" view, that's weird. I was referring to the indicator that this file now doesn't end in a newline.

Comment (digitalcora) [reply .github/dependabot.yml:7]: Screen Shot 2021-07-27 at 5 00 03 PM

**Comment by bklebe** [comment]
> I'm fine with this, but if we were doing a "mass update" (i.e. there were ≥ `limit` PRs open), wouldn't we be doing the package updates locally anyway and thus know all the ones that were outdated?

Also not sure I understand what you're asking here. Do you mean we would run `mix hex.outdated` or the NPM equivalent locally? Not sure how we'd know which are outdated in that case. The benefit of Dependabot seems to be in figuring out, ahead of time and in parallel, which updates pass CI. That and doing it regularly.

File: .github/dependabot.yml (hunk 1 above)
Comment (bklebe) [reply .github/dependabot.yml:7]: Ah. Missed that. Thanks.

**Pull request merged**